devices/debian.py

```diff
@@ -418,6 +511,10 @@ def setup_as_cmts_provisioner(self, board_config):
         self.sendline('/etc/init.d/isc-dhcp-server start')
         self.expect(['Starting ISC DHCP(v4)? server.*dhcpd.', 'Starting isc-dhcp-server.*'])
         self.expect(self.prompt)
+        self.sendline('touch /var/lib/dhcp/dhcpd6.leases')
+        self.expect(self.prompt)
+        self.sendline('/usr/sbin/dhcpd -f -6 -cf /etc/dhcp/dhcpd6.conf -pf /var/run/dhcpd6.pid &')
+        self.expect(self.prompt)
```

Comment:
We can't setup the configuration such that it starts with '/etc/init.d/isc-dhcp-server start' command above?

Reply:
I did try this yesterday. I looked into the file isc-dhcp-server. I'm not sure if modifying isc-dhcp-server is better solution?

Reply:
It's probably just something mis-configured? It would be nice, we do run the restart in a few places to reload configs for example.

Reply:
Correction: I think we need to upgrade our debian nodes and ipv6 should have better support via the initscripts.